meetings/2025-02-05.md

```diff
@@ -28,18 +35,89 @@
 * If this is your first time, welcome! please introduce yourself.
 
 ### Scribes
-* [name]
+* Sebastien Rosset
+* Rui Zhao
 
 ### Introductions
-* [name]
+* Areeba Aziz: From Canada, work at Amazon, really interested in the mission of Solid. Joined to get to know everyone.
 
 ---
 
 ## Topics
 
 ### Update on LWS Use Cases
 
+* HZ: We've collected 100+ use cases, and we are now moving to a second phase of infering requirement out of them. 
+  We will be updating the use cases with a glossary of terms.
+  We came up with a common context for all LWS issues .
+  We'll accept use cases for months to come, so feel free to add more as necessary.
+* eP: Are there are use cases that go beyond the scope of LWS, that would fit more into the CG, like interop ?
+* HZ: IMO absolutely. It is quite probable that the WG is not going to take all requirements.
+  The question translates to what will be in the spec, and the scope of the (WG) group. Maybe we will create a sub-group. And we may ask help to the CG. We will get into this in a couple of months.
+* eP: Will the WG and CG work on the common set of use cases? The requirements could fall either under WG or CG.
+* HZ: The work is public, with issues, etc. So feel free to engage a conversation there.
+* RZ: Some of my use cases don't fit into the current categories. Where should this be discussed ?
+* HZ: You can do that in the issues.
+  Personally I would like to consider Niko's proposal to decouple protocols (HTTP, etc) but I don't know yet if this is going to enter as a requirement for the spec. We're going to discuss this within the WG.
+  Meetings take place every Monday morning US time, have a look at the minutes https://www.w3.org/groups/wg/lws/calendar/?past=1&tf=0
+* Rahul: WG chairs can and is typically expected to invite "experts" for this kind of issues. You can also ask to be invited. 
+* Rahul: You can nominate yourself to be an "invited expert". This may concern Sébastien in particular.
+
+### [New Work Item] ActivityPub/Fediverse interoperability
+
+https://github.com/solid/specification/issues/708
+
+* eP: I will have a call with Sebastien right after this call. We could use few minutes in that call to coordinate next steps.
+* eP: I would like to decide if we take it as a group item. If we create a repo on Friday social web CG will have a meeting. As PAC recommended last week, we should coordinate with the Social Web group.
+* SR: nothing to add, it would be great if a repository could be created.
+* HZ: we can make the decision now. Any objections? (None)
+* Rahul: It's a good idea to formalize that with a repo.
+
+ACTION: elf Pavlik and Hadrian to work with W3C and Jesse to get the solid/activitypub-interop repository created.
```

Comment:
```suggestion
* eP: I will have a call with Sébastien right after this call. We could use few minutes in that call to coordinate next steps.
* eP: I would like to decide whether we take it as a group item. Whether we create a repo. On Friday, Social Web CG will have a meeting. As PAC recommended last week, we should coordinate with the Social Web group.
* SR: Nothing to add. It would be great if a repository could be created.
* HZ: We can make the decision now. Any objections?
  (None)
* Rahul: It's a good idea to formalize that with a repo.

ACTION: elf Pavlik and Hadrian to work with W3C and Jesse to get the [`solid/activitypub-interop`](https://github.com/solid/activitypub-interop) repository created.
```